.github/workflows/lint.yml

```diff
@@ -7,10 +7,12 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
 
 jobs:
-  lint:
+  ruff:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: psf/black@stable
+      - uses: chartboost/ruff-action@v1
```

Comment:
is there a reason why use this one instead of what seems to be the official action https://github.com/astral-sh/ruff ?

Reply:
This comes from their documentation see [this](https://github.com/astral-sh/ruff#usage)